<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>employee_rege</class>
 <widget class="QDialog" name="employee_rege">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="address">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>161</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>587</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>211</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>564</x>
     <y>23</y>
     <width>221</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>97</y>
     <width>48</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>الاسم:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="employee_name">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>112</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>148</y>
     <width>48</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>العنوان</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="employee_id">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>66</y>
     <width>169</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_2">
   <property name="geometry">
    <rect>
     <x>632</x>
     <y>597</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>587</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>67</y>
     <width>11</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>199</y>
     <width>48</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>التليفون</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>277</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>تاريخ الميلاد</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>318</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>رقم البطاقة</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="national_id">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>315</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>275</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>مكان الميلاد</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="POB">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>320</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>تاريخ التوظيف</string>
   </property>
  </widget>
  <widget class="QComboBox" name="blood">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>355</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AB+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AB-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>O+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>O-</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>361</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>فصيلة الدم</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="army_stat">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>معفي من الجيش</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="insurance_stat">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>437</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>تامين</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="appear_as_cashier">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>440</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>كاشير</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="appear_as_seller">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>437</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>بائع</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>370</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>الوظيفة</string>
   </property>
  </widget>
  <widget class="QComboBox" name="job_name">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>365</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="emp_date">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>319</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>405</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>المرتب</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="salary">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>402</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="DOB">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>280</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
